### DISCUSSION TITLE
How can I start to develop?

### DISCUSSION THREAD

**Discussion by oshliaer** [Lednerb/bilberry-hugo-theme#409] [Q&A] [answered]
Hi there!

I see the repo has cool tools and settings for develop. But now I can't how can I start to develop?

As for me I have to run `$> hugo serve --themesDir ./../.. --theme bilberry-hugo-theme` for preview. I think I do something wrong. Is any tutorial or a cheatsheet to start view develop of this theme?

**Comment by igor-baiborodine** [comment, marked as answer]
@contributorpw, have you tried the [Quick Start](https://github.com/Lednerb/bilberry-hugo-theme#quick-start) guide? Or you can use this [tutorial](https://www.kiroule.com/article/start-blogging-with-github-hugo-and-netlify/#create-new-site).

**Reply by oshliaer** [reply]
@igor-baiborodine, thanks!

I have quite successfully published hugo sites on AWS, Google Cloud and Google Firebase hosting, Yandex Cloud.

I mean I want to extend the theme bilberry-hugo-theme.

I'm interested in the approach that is used to start development on the theme itself. For example, I saw the folder with a test environment testing/testing-example-site or webpack.mix.js there. At the same time I see commands to run in package.json/scripts. But I couldn't figure out how to use it.

**Reply by igor-baiborodine** [reply]
Usually, the simplest way to customize the Bilberry theme is to change the look & feel. To do so, you can follow the [Custom Colors and Fonts](https://github.com/Lednerb/bilberry-hugo-theme#custom-colors-and-fonts) section in the README. Here is an example, https://anoff.io/ and the corresponding GitHub [repo](https://github.com/anoff/blog).

Also, please check the [Individual posts customization](https://github.com/Lednerb/bilberry-hugo-theme#individual-posts-customization) section.

Another customization will involve modifying existing HTML partials, such as the [favicon.html](https://github.com/igor-baiborodine/kiroule.com/blob/master/layouts/partials/favicon.html) or [404.html](https://github.com/igor-baiborodine/kiroule.com/blob/master/layouts/404.html) pages:
1. https://github.com/Lednerb/bilberry-hugo-theme#custom-404-page
2. https://github.com/Lednerb/bilberry-hugo-theme#favicons

For more complex customization, you can check this example of how to list posts differently: 
1. https://github.com/igor-baiborodine/bilberry-hugo-theme-sandbox/commit/94769aa1795a80ff325eddaf63f72381483496ce
2. https://deploy-preview-8--bilberry-sandbox.netlify.app/audio/

**Reply by oshliaer** [reply]
Thanks! I see
